### PR TITLE
Code Review 'f/terratest examples' (PR #68)

### PR DESCRIPTION
**Pull request by ThomasObenaus**
This PR is intended to fix some issues from code review for PR #68 (after splitting the testfile).

### CODE REVIEW

**Comment by ThomasObenaus** [comment]
*Hint:* The TestNomadExample works only with the fix provided at 92feaad4a5d84c417c04dd81d0504ceb93766284 (which will be applied with PR #71).

To be able to run the test successfully already to check this PR you have to copy the files [examples/nomad/sg_nomad_consul.tf](https://github.com/MatthiasScholz/cos/pull/71/commits/92feaad4a5d84c417c04dd81d0504ceb93766284#diff-846291a2f826697481e001c535976d49) and [examples/nomad/sg_consul_nomad.tf](https://github.com/MatthiasScholz/cos/pull/71/commits/92feaad4a5d84c417c04dd81d0504ceb93766284#diff-8d40f689fea98aa85eda0348770e21a1) to your local `examples/nomad` folder.

File: examples/ui-access/outputs.tf [hunk 1]
```diff
@@ -31,5 +31,5 @@ output "curl_fabio_ui" {
 }
 
 output "url_fabio_ui" {
-  value = "http://${module.ui-access.fabio_ui_alb_dns_name}/"
+  value = "http://${module.ui-access.fabio_ui_alb_dns_name}/health"
```
[the remark this answers is not in the document]
Comment (ThomasObenaus) [reply examples/ui-access/outputs.tf]: It was my fault. When I asked why there is this `/health` path, I thought the code was part of the module and not of the example.

The `/health` is needed to be able to deploy the ALB example and be able to test it. In this example beside the ALB also a EC2 instance is deployed. On this instance runs an `httpd` instance providing a 200_OK response at `/health` (see: https://github.com/MatthiasScholz/cos/blob/master/examples/ui-access/user-data.sh#L35).

Without the adjustment above also the `TestUIAccessExample` fail for the fabio check.

**Comment by ThomasObenaus** [comment]
> Could you please provide some further insides about why major changes were made.

Can you give me a hint for which change you want to know more?

**Comment by MatthiasScholz** [comment]
Additionally a nomad test was added.

**Pull request merged**